lib/selects.bzl

```diff
@@ -234,11 +234,11 @@ def _config_setting_always_true(name, visibility):
     name_off = name + "_stamp_binary_off_check"
     native.config_setting(
         name = name_on,
-        values = {"stamp": True},
+        values = {"stamp": "1"},
     )
     native.config_setting(
         name = name_off,
-        values = {"stamp": False},
+        values = {"stamp": "0"},
```

Comment:
I won't argue about the correctness of this, but it does seem incredibly weird that True and False don't combine to cover all values, but the string values "0" and "1" do.

Maybe I'm just grumbly. But IMO, converting ints to bools is OK from historical perspective, but strings should not.